Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet/ModulePages/AnalyticsContentPage.xaml

```diff
@@ -5,7 +5,7 @@
              x:Class="Contoso.Forms.Puppet.AnalyticsContentPage">
     <TableView Intent="Form">
         <TableSection Title="Analytics Settings">
-            <SwitchCell Text="Analytics Enabled" On="true" x:Name="EnabledSwitchCell" OnChanged="UpdateEnabled" />
+            <SwitchCell Text="Analytics Enabled" On="true" x:Name="EnabledSwitchCell" OnChanged="UpdateEnabled" OnColor="{StaticResource SwitchCellOnColor}"/>
```

Comment:
<img width="1335" alt="Screen Shot 2019-11-15 at 11 37 53 AM" src="https://user-images.githubusercontent.com/623672/68970549-650f1000-079c-11ea-955c-3846a2fc833d.png">

These color changes are out of scope for the work item and causing crashes.

Reply:
Remove these color changes.

Reply:
It definitely crashes even after a git clean.